tomography_preprocessing/tilt_series_alignment/aretomo/batch.py

```diff
@@ -62,7 +65,7 @@ def batch_aretomo(
             n_patches_xy=n_patches_xy,
             alignment_thickness_px=alignment_thickness,
             tilt_angle_offset_correction=tilt_angle_offset_correction,
-            output_directory=output_directory,
+            job_directory=external_directory / tilt_series_id,
```

Comment:
This line causes align_tilt_series in aretomo to create another tilt series and alignments directory within external/[Tomo_Name]/ and saves the star file/stack there instead of in the main job directory tilt_series and alignments directory.

Reply:
I think this should be: 
`job_directory=output_directory`
Testing now :)

Reply:
Yes, this is the issue and the correct fix :)